tests/test_model/test_fast_conv_bn_eval.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from torch import nn
+
+import mmengine.model.fast_conv_bn_eval as mod
+from mmengine.model.fast_conv_bn_eval import \
+    turn_on_fast_conv_bn_eval_for_single_model
+from mmengine.testing import assert_allclose
+
+optimize_conv_module = mod.optimize_conv_module
+
+
+class BackboneModel(nn.Module):
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        global optimize_conv_module
+        if optimize_conv_module:
+            from mmcv.cnn import ConvModule
+            conv0 = nn.Conv2d(6, 6, 6)
+            bn0 = nn.BatchNorm2d(6)
+            self.mod1 = ConvModule.create_from_conv_bn(conv0, bn0)
+        self.conv1 = nn.Conv2d(6, 6, 6)
+        self.bn1 = nn.BatchNorm2d(6)
+        self.conv2 = nn.Conv2d(6, 6, 6)
+        self.bn2 = nn.BatchNorm2d(6)
+        self.conv3 = nn.Conv2d(6, 6, 6)
+        self.bn3 = nn.BatchNorm2d(6)
+
+    def forward(self, x):
+        global optimize_conv_module
```

Comment:
same as above

Reply:
My bad. Now I remember that `global` is used to modify global variables. These lines can be removed. Fixed in [bee4711](https://github.com/open-mmlab/mmengine/pull/1202/commits/bee47111a16e086b2e666c3868c25a6262cf1d79).